envi/archs/arm/__init__.py

```diff
@@ -11,9 +11,11 @@
 class ArmModule(envi.ArchitectureModule):
 
     def __init__(self, name='armv6'):
+        import envi.archs.thumb16.disasm as eatd
```

Comment:
these changes allow for arm to easily switch to thumb mode based on VA.  most binary containers for arm list entry points as odd VA's for thumb mode, even though the actual instructions start at the even addresses.